packages/typespec-python/tasks.py

```diff
@@ -29,7 +29,10 @@
         "package-name": "azure-hello",
         "package-mode": "dataplane",
         "package-pprint-name": "Hello",
-    }
+    },
+    "lro-basic": {"package-name": "azure-lro-basic"},
```

Comment:
do we need to change the package name?

Reply:
`lro-basic` has namespace `azure.lro` which is conflicted with `lro-rpc`/`lro-core` that has namespace `azure.lro.rpc`/`azure.lro.core`. And it also looks unified with prefix `azure-lro-XX`